flink-connectors/flink-connector-hive/pom.xml

```diff
@@ -59,6 +59,12 @@ under the License.
 			<version>${project.version}</version>
 			<scope>provided</scope>
 		</dependency>
+                <dependency>
```

Comment:
the indentation looks not right. Not sure if it's a problem of github. Can you double check?